app/views/notices/shop_employee_notices/employee_dependent_termination_notice.html.erb

```diff
@@ -0,0 +1,42 @@
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+																									content: "Dependent children with disabilities may be eligible to remain covered by their parents’ policy regardless of their age. Please contact the #{site_short_name} at #{contact_center_phone_number} regarding adult children with disabilities that need to remain enrolled on an employee’s plan beyond age 26 or if you believe this decision is the result of an error."} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+																									content: "<strong>Individual Market Enrollment</strong><br>", col_styles: ['margin-bottom: 0px']} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+																																		 content: "If your dependent child does not qualify to enroll in employer-sponsored coverage or other coverage at this time, they can enroll during annual individual market open enrollment period. During this period your dependent can enroll in individual market coverage through the #{site_short_name} if they live in Massachusetts.  Your dependent child may also be eligible for premium subsidies and other forms of financial assistance if they are unable to afford the full cost of health insurance.  Your dependent child can create a #{site_short_name} account and complete an eligibility application for individual marketplace coverage to review health plan options and learn whether they might be eligible for financial assistance. If your dependent lives outside of Massachusetts, they can apply during open enrollment through the health insurance exchange operating in their state. To find the Health Insurance Marketplace in your state, go to <a href= '#{health_care_website_url}'> #{health_care_website}. </a> "} %>
+
+<%= render partial: 'notices/shared/paragraph', :locals => {
+																									content: "There are separate eligibility rules for Special Enrollment Periods into individual market coverage.  If your dependent is a Massachusetts resident and thinks they might be eligible for an individual market Special Enrollment Period, they can contact the #{site_short_name} at #{contact_center_ivl_phone_number} to see if they are eligible to enroll through a Special Enrollment Period."} %>
+
+<%= render partial: 'notices/shared/paragraph_with_headers', :locals =>
+																															 { heading: 'For Questions or Assistance:',
+																																 col_styles: ['margin-bottom: 0px']} %>
+<%= render partial: 'notices/shared/paragraph', :locals =>
+																									{ content: "Please contact your benefits office for further assistance. You can also contact the #{site_short_name} with any questions:"} %>
+
+<%= render partial: 'notices/shared/unordered_list', :locals => { listings:
+                                                                      ["By calling #{contact_center_phone_number}.","TTY: #{contact_center_tty_number}", "Online at: #{site_website_address}" ] } %>
```

Comment:
Online at site link is not working please do check it and mention it as MAhealthconnector.org.

Reply:
It's my bad. It hasn't reloaded in my system. it's working fine.

Reply:
Checked it twice. It works fine for me. Please make sure you are checking in the right PDF.

Reply:
Checked it twice. works fine for me. Please make sure you are checking the right PDF file.